x-pack/legacy/plugins/ml/public/services/ml_api_service/index.js

```diff
@@ -444,6 +444,13 @@ export const ml = {
     });
   },
 
+  esSearchRx(obj) {
```

Comment:
Should we name this `esSearch$`?

Reply:
I agree with this naming convention - should make it super clear when we're working with/expecting observables.

Reply:
done in https://github.com/elastic/kibana/pull/51008/commits/c72dddf5ed6d8184db3ed0ea95cc1850b83ab701